src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -172,7 +172,6 @@ public sealed partial class ComAliasNameAttribute : System.Attribute
         public string Value { get { throw null; } }
     }
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-    [System.ObsoleteAttribute("ComAwareEventInfo may be unavailable in future releases.")]
```

Comment:
Were these only in the refs rather than also being in the implementation?

Reply:
I think so. Should they have been on both?

Reply:
Generally they would have been.  I'm sure we have other inconsistencies in that regard, though.